package.json

```diff
@@ -29,16 +29,21 @@
     "window-or-global": "1.0.1"
   },
   "devDependencies": {
+    "@types/incremental-dom": "^0.5.0",
```

Comment:
do we need those? since skate will not use TS internally? I mean we don't need this for our typings defs right?